goldmane/pkg/daemon/daemon.go

```diff
@@ -66,14 +66,23 @@ type Config struct {
 	PushIndex int `json:"push_index" envconfig:"PUSH_INDEX" default:"30"`
 }
 
-func Run() {
+func ConfigFromEnv() Config {
 	// Load configuration from environment variables.
 	var cfg Config
 	if err := envconfig.Process("", &cfg); err != nil {
 		logrus.WithError(err).Fatal("Failed to load configuration from environment")
 	}
 
 	utils.ConfigureLogging(cfg.LogLevel)
+
+	return cfg
+}
+
+func (cfg Config) ConfigureLogging() {
```

Comment:
this function is still here @Brian-McM

Reply:
Whoops, what did I previously delete lol.

Reply:
I think you're looking at the old commit @caseydavenport